tests/sentinel/tests/includes/notify.sh

```diff
@@ -1,17 +1,26 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 # fd 3 is meant to catch the actual access to /proc/pid/fd, 
 # in case there's an fd leak by the sentinel,
 # it can take 3, but then the access to /proc will take another fd, and we'll catch that.
-leaked_fd_count=`ls /proc/self/fd | grep -vE '^[0|1|2|3]$' | wc -l`
+
+OS=`uname -s`
 
 sentinel_fd_leaks_file="../sentinel_fd_leaks"
 
-if [ $leaked_fd_count -gt 0 ]
+if [ ${OS} == "Linux" ]
 then
-    if [ ! -f $sentinel_fd_leaks_file ]
+    leaked_fd_count=`ls /proc/self/fd | grep -vE '^[0|1|2|3]$' | wc -l`
+    if [ $leaked_fd_count -gt 0 ]
     then
-        ls -l /proc/self/fd | cat >> $sentinel_fd_leaks_file
+        if [ ! -f $sentinel_fd_leaks_file ]
+        then
+            ls -l /proc/self/fd | cat >> $sentinel_fd_leaks_file
+        fi
     fi
+else
+    echo "OS: expected Linux, but got ${OS}."
```

Comment:
Sorry. My bad for completely forgetting there are other OSes. 

We have quite a few places that use `uname` in the TCL code to skip tests. 
So we can skip registering that bash script. (instead of the change in the bash file).
 
I don't have an objection for that change in the bash script, but I rather an early exit than the indentation you added. 
And I don't think the `echo` is useful, I rather drop it. 

I'll look into the CI issue..

Reply:
It seems to make a little more sense to do this in TCL rather than bash.

Since this PR is already merged in, then I just open a new PR for it?

Reply:
https://github.com/redis/redis/pull/8364